syntax = "proto3";

package PbData;

message RepeatedValueUint64 { repeated uint64 values = 1; }
message RepeatedValueDouble { repeated double values = 1; }

message ValuesMap {
  RepeatedValueUint64 timestamp = 1;
  map<string, RepeatedValueDouble> valuesMap = 2;
}

message TimeValuesPack { map<string, ValuesMap> valuesPack = 1; }
